acceptance/lib/pxp-agent/bolt_pxp_module_helper.rb

```diff
@@ -81,13 +85,27 @@ def run_task(broker, targets, task, files, input: {}, metadata: nil, **kwargs, &
   do_module_action(broker, agent_identities(targets), 'task', 'run', params, **kwargs, &block)
 end
 
+def script_request(broker, agent, script, arguments, **kwargs, &block)
+  params = { arguments: arguments, script: script }
+  target = ["pcp://#{agent}/agent"]
+  do_module_action(broker, target, 'script', 'run', params, **kwargs, &block)
+end
+
 # Makes a non-blocking request to start a command.
 # Block is executed on the "response_dataset" object from do_module_action.
 def run_command(broker, targets, command, **kwargs, &block)
   params = { command: command }
   do_module_action(broker, agent_identities(targets), 'command', 'run', params, **kwargs, &block)
 end
 
+# Starts a download file execution.
```

Comment:
A couple functions moved around in this file (to make it easier to read), but have not been changed:

run_download_file
ensure_failed